llumnix/config/default.py

```diff
@@ -112,7 +112,7 @@
 # Number of kv-cache layers to transfer in each round during migration
 _C.MANAGER.MIGRATION_NUM_LAYERS = 1
 # Number of internal cache size in migration backend for sending and receiving
-_C.MANAGER.MIGRATION_INTERNAL_BUFFER_NUM = 2
+_C.MANAGER.MIGRATION_INTERNAL_BUFFER_NUM = 1
```

Comment:
why?

Reply:
Currently, Llumnix is still unstable and migration test is still have high possibility of failure. So I think we can disable the many-to-many migrations temporarily until our migration tests are stable.

Reply:
set MIGRATION_INTERNAL_BUFFER_NUM to 2 will cause ci to fail ?

Reply:
Unsure, I think there are serveral bugs make our migration tests failed, which are still need to be figured out.